torchaudio/functional/functional.py

```diff
@@ -1631,3 +1632,35 @@ def rnnt_loss(
         return costs.sum()
 
     return costs
+
+
+def compute_rtf_power(
+    psd_s: Tensor, psd_n: Tensor, reference_channel: Union[int, Tensor], iterations: int = 3
+) -> Tensor:
+    r"""Estimate the relative transfer function (RTF) or the steering vector by the power method.
+    Args:
+        psd_s (Tensor): The complex-valued covariance matrix of target speech.
+            Tensor of dimension `(..., freq, channel, channel)`
+        psd_n (Tensor): The complex-valued covariance matrix of noise.
+            Tensor of dimension `(..., freq, channel, channel)`
+        reference_channel (int or Tensor): Indicate the reference channel.
+            If the dtype is ``int``, it represent the reference channel index.
+            If the dtype is ``Tensor``, the dimension is `(..., channel)`.
+        iterations (int): number of iterations in power method. (Default: ``3``)
+    Returns:
+        Tensor: the estimated complex-valued RTF of target speech
+            Tensor of dimension `(..., freq, channel)`
+    """
+    phi = torch.linalg.solve(psd_n, psd_s)  # psd_n.inv() @ psd_s
+    if torch.jit.isinstance(reference_channel, int):
+        rtf = phi[..., reference_channel]
+    else:
+        rtf = torch.einsum("...c,...c->...", [phi, reference_channel[..., None, None, :]])
+    rtf = rtf.unsqueeze(-1)  # (..., freq, channel, 1)
+    if iterations >= 2:
+        for _ in range(iterations - 2):
```

Comment:
Can you add a comment of why `- 2` so that future maintainers who do not have knowledge of this algorithm knows it's indented and won't suspect it's a bug?